src/components/DashboardLayout/NotificationPopover.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  faBell,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Popover } from '@headlessui/react';
+import { NotificationContext } from 'data/NotificationContext';
+
+import NotificationPanel from './NotificationPanel';
+import { useContext, useEffect, useState } from 'react';
+import clsx from 'clsx';
+
+export const NotificationPopover = () => {
+  const { notifications, ongoingNotifications } = useContext(NotificationContext);
+  const [ newNotifications, setNewNotifications ] = useState<boolean>(false);
+
+  useEffect(() => {
+    setNewNotifications(notifications.length > 0 || ongoingNotifications.length > 0);
```

Comment:
when the page first loads, "newNotification" is set to true, it should be false

currently, the bell gets rendered first, then notifications are fetched and added to the list one by one, causing the bell to light up.

when notifications are added to the `NotificationContext`, they are marked either stale or fresh, we only want the bell to light up for fresh notifications.

We might need to edit NotificationContext to achieve this

Reply:
`newNotifications` is set to `false` on page load, not sure what you mean.
where are they marked stale/fresh? `OngoingNotificationItem`s have a `state` field but this just whether it's ongoing or finished

Reply:
sorry I wasn't very clear:
Practically, `newNotifications` will be true shortly after the page loads.

On page load, `EventStream` tries to fetch all history notifications and "replay" them, that's how we are able to see old notifications (not live from websocket).
This causes `newNotifications` to become true. 

In `EventStream`, in `handleEvent`, events are marked as "fresh" or not https://github.com/Lodestone-Team/lodestone/blob/fb96ec54c8b9d1279eefb8bad8b0ee554e57925b/src/data/EventStream.ts#L93

We would like the bell to only light up when a new "fresh" notification comes in

Reply:
added `fresh` to the `NotificationItem` definition to fix this